ONSdigital/eq-questionnaire-runner
# Fix unrelated block routing when question variants exist

### What is the context of this PR?
Fixes a bug with routing from the "No" answer for the "Related to anyone else" question when variants exist with different "No" answer values. Before this fix the "No" answer wasn't skipping the remaining relationships for that person.

### How to review 
- Use the `test_relationships_unrelated` schema
- Add six people
- Answer unrelated, unrelated, related to anyone else no, unrelated, unrelated, related to anyone else no
- You should be taken to the next persons relationships questions i.e. skip the rest of the relationship questions for that person

This should also be tested in the census schemas.

### Checklist
